docs/src/pages/components/data-grid/overview/DataGridDemo.tsx

```diff
@@ -3,13 +3,27 @@ import { DataGrid, GridColDef, GridValueGetterParams } from '@material-ui/data-g
 
 const columns: GridColDef[] = [
   { field: 'id', headerName: 'ID', width: 90 },
-  { field: 'firstName', headerName: 'First name', width: 150 },
-  { field: 'lastName', headerName: 'Last name', width: 150 },
+  {
+    field: 'firstName',
+    headerName: 'First name',
+    width: 150,
+    editable: true,
+    disableClickEventBubbling: true,
```

Comment:
don't we need `disableSelectionOnClick` here as well?

Reply:
I missed this one.